third-party/realdds/include/realdds/topics/control/control-msg.h

```diff
@@ -17,36 +17,36 @@ class control
     using type = raw::device::controlPubSubType;
 
     //Open uses a vector of profiles, to have constant message size we limit it
-    static const size_t MAX_OPEN_PROFILES = 8;
+    static const size_t MAX_OPEN_STREAMS = 8;
 
 #pragma pack( push, 1 )
-    enum class control_type : uint16_t
+    enum class msg_type : uint16_t
     {
-        SENSOR_OPEN,
-        SENSOR_CLOSE,
+        STREAMS_OPEN,
+        STREAMS_CLOSE,
         MAX_CONTROL_TYPE
     };
 
     struct stream_profile
     {
+        int16_t uid;             // Stream unique ID
         int16_t framerate;       // FPS
-        rs2_format format;       // Transfer as uint8_t?
-        rs2_stream type;         // Transfer as uint8_t?
+        int8_t format;           // Corresponds to rs2_format
+        int8_t type;             // Corresponds to rs2_stream
         int16_t width;           // Resolution width [pixels]
         int16_t height;          // Resolution width [pixels]
     };
 
-    struct sensor_open_msg
+    struct streams_open_msg
     {
         uint32_t message_id; // Running counter
-        int16_t sensor_uid;  // Sensor ID as received in notification::sensor_header_msg.index
-        stream_profile profiles[MAX_OPEN_PROFILES];
+        stream_profile streams[MAX_OPEN_STREAMS];
     };
 
-    struct sensor_close_msg
+    struct streams_close_msg
     {
         uint32_t message_id; // Running counter
-        int16_t sensor_uid;
+        int16_t stream_uids[MAX_OPEN_STREAMS];
```

Comment:
Are the uids enough?

Reply:
Oh wait, these are stream uids? Not profile? What's a stream uid?

Reply:
Profile uid field actually carries stream uid (all profiles of same stream have same uid filed)
We need to think about it a bit more, if streams will be uniquely identified by uid+index pairs or maybe stream server will build a stream ID comprised of the two.
As discussed, stream will have to send their names, but I don't think names are good enough IDs

Reply:
>all profiles of same stream have same uid filed)
Really? Depth/Infrared1/2/Confidence all have the same uid? They differ in their index?

Reply:
Streams will be identified in the API by the object.
In the communication, the name is unique: we cannot have two with the same name. I see no issue with identifying them that way. Let's try that (not in this PR) and see if we want to also include a stream ID (set by the server, of course).

Reply:
Depth/Infrared are streams of same sensor. Depth have different ID than IR, but IR1 and IR2 have same ID, differ by index.
All profiles of IR1 have same UID+index pair, all depth stream profiles have same UID, etc...

Reply:
How does the server know how many UIDs were passed? It assumes a 0 is the end? So we cannot use a UID of 0?
I think we need to think about this.
Not for this PR.

Reply:
I'm almost inclined to communicate json strings...

Reply:
Added number_of_streams field to the message